libs/statistics-popover/src/components/statistics-info/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { KILOBYTES, MEGABYTES } from './constants';
+
+export const formatBitrate = (bitrate: number): string => {
+  return `${formatNumber(bitrate)}bps`;
+};
+
+export const formatBytes = (bytes: number): string => {
+  return `${formatNumber(bytes)}B`;
+};
+
+export const formatNumber = (input: number): string => {
```

Comment:
`formatNumber` is too generic,  if the function name couldn't describe itself, please add a comment.